components/event-bus/internal/knative/util/kn-eventing.go

```diff
@@ -64,7 +64,7 @@ var once sync.Once
 type KnativeAccessLib interface {
 	GetChannel(name string, namespace string) (*messagingV1Alpha1.Channel, error)
 	GetChannelByLabels(namespace string, labels *map[string]string) (*messagingV1Alpha1.Channel, error)
-	CreateChannel(name string, namespace string, labels *map[string]string,
+	CreateChannel(prefix string, namespace string, labels *map[string]string,
```

Comment:
maps are pointers, so `*map` is a big no-no.
Redundant `string`.
```suggestion
	CreateChannel(prefix, namespace string, labels map[string]string,
```